docs/en/development/build.md

```diff
@@ -140,6 +140,6 @@ hash cmake
 
 ClickHouse is available in pre-built binaries and packages. Binaries are portable and can be run on any Linux flavour.
 
-They are built for stable, prestable and testing releases as long as for every commit to master and for every pull request.
+They are built for stable and lts releases as long as for every commit to master and for every pull request.
```

Comment:
Should this say:
Binaries are built for stable and LTS releases and also every commit to `master` and for each pull request.

Reply:
Thanks and I use this recommended comment :).